nvflare/apis/aux_spec.py

```diff
@@ -27,15 +27,14 @@ class MessageSendStatus(enum.Enum):
 
 
 def aux_request_handle_func_signature(topic: str, request: Shareable, fl_ctx: FLContext) -> Shareable:
-    """
-    This is the signature of the message_handle_func.
+    """This is the signature of the message_handle_func.
```

Comment:
We will need to improves this.
What is a message_handle_func?

Reply:
I believe that is a function that gets registered for the aux message handler. I agree this isn't very clear, I can add more detail.

Reply:
@SYangster , one way to add more details is to say "any implementation of message_handle_func has to follow this signature.  message_handle_func is a callback function to handle aux request."  You can also use this as an example so users can know more about this signature (engine.register_aux_message_handler(topic=ReservedTopic.END_RUN, message_handle_func=self._handle_end_run))